test/Sdk.Generator.Tests/FunctionMetadataProviderGeneratorTests/IntegratedTriggersAndBindingsTests.cs

```diff
@@ -119,6 +132,19 @@ public Task<ImmutableArray<IFunctionMetadata>> GetFunctionMetadataAsync(string d
                                 ScriptFile = "TestProject.dll"
                             };
                             metadataList.Add(Function0);
+                            var Function1RawBindings = new List<string>();
```

Comment:
In the json generator, when the return type does not include a property for the `HttpResponse` output binding, it is not generated. Adding this test case for sourcegen as well.